src/storage-area.cpp

```diff
@@ -78,6 +79,10 @@ OrthancPluginErrorCode WriteDicomFile(DicomFile dicom, const char *uuid){
                 // todo: probably a better error code
                 return OrthancPluginErrorCode_EmptyRequest;
             }
+
+            // Compute MD5
+            DicomChecksum::calc_checksum(uuid, content.get(), size);
```

Comment:
This will need modification when merged with #106 the `WriteDicomFile()` function no longer exists. It could be added to the Create callback, or possibly to the `DicomFile::Write()` method.